flink-end-to-end-tests/flink-stream-sql-test/src/main/java/org/apache/flink/sql/tests/StreamSQLTestProgram.java

```diff
@@ -343,4 +373,50 @@ public void restoreState(List<Integer> state) {
 		}
 	}
 
+	private static StreamTableEnvironment createStreamTableEnvironment(
```

Comment:
This is unnecessary. Just use the `StreamTableEnvironment.create(sEnv, settings)`

Reply:
Agree